### DISCUSSION TITLE
new gapple health bar and rating system

### DISCUSSION THREAD

**Discussion by Peety07** [JordanSantiagoYT/FNF-JS-Engine#654] [Ideas]
I would like to see you try this, and also I want to have the true gapple hud (icon bop is there and the timer can be recreated by setting time bar to dave engine and time text to modern time and no bar.) Btw this is what I'm referring to for the rating system
https://www.youtube.com/watch?v=xwyfzmkr_uY

**Comment by JordanSantiagoYT** [comment]
1. gapple hud already exists lol (icon bop and timer are already there, health bar is just the vanilla healthbar style option)
2. a similar version of the rating popups seen in that video already exists (Set Rating Type to "Simple" in the current version, OR turn on "Simple Rating Popups" in 1.36.0 or newer)

**Comment by Peety07** [comment]
maybe you could add them in once gapple 1.5 releases

**Reply by moxie-coder** [reply]
we already stated they already exist
